### ISSUE TITLE
a11y: Package release table

### ISSUE THREAD

**Issue by woodruffw** [pypa/warehouse#6010]
Minor accessibility improvements can be made to the release table on `/project/{package}/releases/`:

* The `<table>` itself should have a `<caption>`, as the first child node. This is partially mitigated by the `<h2>` above
* The `<table>` should include `scope=col` or another appropriate scope for each `<th>` node

**Comment by nlhkabu** [comment]
@woodruffw which page are you talking about here?  I can't find it.

For example, both of these URLs are returning a 404:

- https://pypi.org/project/Django/2.2.3/releases/
- https://pypi.org/project/Django/releases/

**Comment by di** [comment]
@nlhkabu I think he's referring to the logged-in view, e.g.:

`/manage/project/{project name}/releases/`

**Comment by woodruffw** [comment]
Yep, I meant under `/manage/`, my apologies.

**Comment by nlhkabu** [comment]
I think this is therefore a duplicate of https://github.com/pypa/warehouse/issues/6079
@woodruffw please reopen if I am incorrect.

**Comment by woodruffw** [comment]
Yes, looks like it. I could have sworn that I created two different issues for the public and management release tables, but looks like I duplicated them.
